pygrep/__init__.py

```diff
@@ -2,78 +2,129 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from __future__ import absolute_import
-
+import argparse
 import ast
+import os.path
 from collections import deque
+from collections.abc import Generator
 from os import walk
-import os.path
-from six.moves import zip
 
 
-class IdentVisitor(ast.NodeVisitor):
-    def __init__(self, path, ident, callback):
+class IdentVisitor:
+    def __init__(self, ident: str):
         super(IdentVisitor, self).__init__()
```

Comment:
In so much as you want to keep this around, this fits into a 2->3 conversion pattern.  I think technically _everything_ should super init, but we usually don't bother...

```suggestion
        super().__init__()
```